yang/ietf-l3vpn-ntw.yang

```diff
@@ -2345,6 +2353,38 @@ module ietf-l3vpn-ntw {
                     if-feature "vpn-common:bfd";
                     description
                       "Container for BFD.";
+                    leaf transmit_interval {
+                      type uint32;
+                      units microseconds;
```

Comment:
RFC5880 says the following: 

"This MUST be
      initialized to a value of at least one second (1,000,000
      microseconds)". 

A default statement with this value should be added.